### PR TITLE
Remove database settings in local_settings

### PR DESCRIPTION
**Pull request by zpallin**
Database settings will need to match a local database

### CODE REVIEW

File: src/project/local_settings.py.template [hunk 1]
```diff
@@ -25,6 +25,7 @@ SHAREABOUTS = {
 MAPQUEST_KEY = 'Fmjtd%7Cluur2g0bnl%2C25%3Do5-9at29u'
 
 # API settings
+"""
```
Comment (wilsonww) [inline src/project/local_settings.py.template]: This is wrong

Comment (zpallin) [inline src/project/local_settings.py.template]: No it's not

**Comment by jainu11** [comment]
It works

File: src/project/local_settings.py.template (hunk 1 above)
Comment (dtahiliani) [inline src/project/local_settings.py.template]: wilson is wrong

File: src/project/local_settings.py.template [hunk 2]
```diff
@@ -35,6 +36,7 @@ DATABASES = {
         'PORT': '5432',
     }
 }
+"""
```
Comment (jainu11) [inline src/project/local_settings.py.template]: Here it is..

**Pull request merged**